src/patternfly.module.js

```diff
@@ -7,9 +7,10 @@
 angular.module('patternfly', [
   'patternfly.autofocus',
   'patternfly.card',
+  'patternfly.charts',
```

Comment:
Ok, so should I update the angular-patternfly README.md specifying what one has to do to use the charts?